yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -193,8 +193,13 @@ class MapOperator : public FlowableOperator<U, D, MapOperator<U, D, F>> {
         : SuperSubscription(std::move(flowable), std::move(subscriber)) {}
 
     void onNext(U value) override {
-      auto&& map = SuperSubscription::getFlowableOperator();
-      SuperSubscription::subscriberOnNext(map->function_(std::move(value)));
+      try {
+        auto&& map = this->getFlowableOperator();
+        this->subscriberOnNext(map->function_(std::move(value)));
+      } catch (const std::exception& exn) {
+        folly::exception_wrapper ew{std::current_exception(), exn};
```

Comment:
I believe we need to have this try/catch at the base Subscriber.
If we receive an exception when onNext is called, it should be caught and onError should be called.

Reply:
I did not need any change in the parent class, Subscriber. The changes in here solved an issue in my tests, that is: "throw at onNext call".
D5952697 - the unit tests with name InputThrows and ResultThrows were not passing before adding this try/catch.